pkg/query/querier.go

```diff
@@ -439,23 +430,23 @@ func (q *querier) LabelValues(name string, matchers ...*labels.Matcher) ([]strin
 		return nil, nil, errors.Wrap(err, "proxy LabelValues()")
 	}
 
-	var warns storage.Warnings
+	var warns annotations.Annotations
 	for _, w := range resp.Warnings {
-		warns = append(warns, errors.New(w))
+		warns.Add(errors.New(w))
 	}
```

Comment:
Can also just use `warns := annotations.New().Merge(resp.warnings)`

Reply:
No, storepb APIs return []string for warnings still